docs/guides.md

```diff
@@ -21,6 +22,11 @@ echo 'xenguest_enable="YES"' >> /etc/rc.conf.local
 ln -s /usr/local/etc/rc.d/xenguest /usr/local/etc/rc.d/xenguest.sh
 service xenguest start
 ```
+Option 2 is via webgui:  
+Open management page under http(s)://your-configured-ip and go to:  
```

Comment:
Use Markdown to get a bullet or ordered list, it would be easier to read :+1:

Reply:
Honestly I don't really know what you want, that it's "easier to read". I like the -> -> -> style, as it shows Step by step what to do. Though it's not a list of points to walk down, it's just the way to get where you need to go.

Reply:
No I was thinking:

"You need to do this:
* first thing
* then that
* finally this
"

A bullet list.